### PR TITLE
docs(backup-restore): update velero backup docs

### PR DESCRIPTION
**Pull request by sinhaashish**

### CODE REVIEW

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ aws_secret_access_key = minio123
 Install Velero using the command below:
 
 ```
-velero install --provider aws --bucket velero --secret-file /home/pawan/velero/credentials-minio --plugins velero/velero-plugin-for-aws:v1.0.0-beta.1 --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000 --use-volume-snapshots=true --use-restic
+velero install --provider aws --bucket velero --secret-file /home/pawan/velero/credentials-minio --plugins velero/velero-plugin-for-aws:v1.0.0-beta.1 --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000 --use-volume-snapshots=true --use-node-agent --uploader-type restic
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md:50]: Are these the changes needed to make this work with the latest Velero?

If we use the restic way, does it work for block mode of volumes as well?

Comment (sinhaashish) [reply docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md:50]: yes , tested it against the latest velero. `--use-restic` flag was deprecated long ago and installation fails if we use `--use-restic` flag.

Comment (avishnu) [reply docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md:50]: What about the block mode of volumes?

Comment (sinhaashish) [reply docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md:50]: yes, tested it against the block volumes as well using this docs https://github.com/openebs/zfs-localpv/blob/develop/docs/raw-block-volume.md

**Review by avishnu — APPROVED** [review]

**Pull request merged**